networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/rpc/beaconchain/methods/BeaconBlocksByRangeMessageHandler.java

```diff
@@ -45,6 +46,10 @@
   static final RpcException INVALID_STEP =
       new RpcException(INVALID_REQUEST_CODE, "Step must be greater than zero");
 
+  private static final RpcException TOO_MANY_BLOCKS =
+      new RpcException(
+          INVALID_REQUEST_CODE, "Only a maximum of " + MAX_REQUEST_BLOCKS + " can per request");
```

Comment:
Anton moved us away from using static instances of `RpcException`.  Might need to pull in the latest master to get those changes.

Reply:
oh misread, just inline this so we create the exception when we pass it to completeWithErrorResponse.